Makefile

```diff
@@ -51,6 +51,10 @@ check_flake8:
 check_isort:
 	tox -e isort -- --check $(CHECK_SYNTAX_FILES)
 
+## works the same as .github actions checks
+postgres:
+	docker run -itd --rm --name dab_postgres --net=host -e POSTGRES_USER=dab -e POSTGRES_PASSWORD=dabing -e POSTGRES_ADMIN_PASSWORD=dabing -e POSTGRES_DATABASE=dab_db postgres:13
```

Comment:
From the comment on this block I reckon this follows some existing pattern.

I would have expected this to use the existing `docker-compose` commands to start the existing `postgres` service and use that database service/container to run the tests against.

Reply:
There are no existing `docker-compose` commands, and that doesn't mean I _want_ them either.

test_app is a beautiful thing, because it runs locally. It does its job well, which is to help test things. I don't want to add container wrappers, in part, because I don't want to maintain a test_app image. I would rather spend time making DAB runnable with fewer dependencies (ping @TheRealHaoLiu), which adds additional value, than creating a new image that no one wants.

The value that `make postgres` provides is that it runs a postgres container with the same postgres config as what the github actions do (username, password, etc). That means you can use the same Django settings for both. I looked into sharing an env file, and it doesn't seem to be possible with the github actions service pattern. Now `make postgres` will start a container in the background, which I justify by looking at the eda-server `task docker:up:minimal`. This is very different from the AWX (and others) workflow, but I believe it better matches DAB which is, and should be, a library first and foremost.

Reply:
:facepalm: I thought this was the gateway repo.

:+1: